viewer/packages/360-images/src/entity/Image360Entity.ts

```diff
@@ -55,15 +55,15 @@ export class Image360Entity implements Image360 {
     this._transform = transform;
     this._image360Icon = icon;
     this._image360VisualzationBox = new Image360VisualizationBox(this._transform, sceneHandler);
+    this._image360VisualzationBox.visible = false;
```

Comment:
Just so I understand - is this just a general fix, and not really related to aborting requests?